### DISCUSSION TITLE
Enable Dark Mode

### DISCUSSION THREAD

**Discussion by bloodshinoda** [eneshecan/whatsapp-for-linux#255] [Q&A] [answered]
I know the browser part of the application can be in dark mode, but there isn't an easy fix for the headerbar as of now...

I don't remember the workaround on the .desktop file to force dark mode...

What is the best possible solution for that?

Using Manjaro Gnome here...

**Comment by xeco23** [comment]
If I understand you correctly, this is something that I was working on recently to put a dark theme option (Prefer Dark Theme) into the preferences window. I will push it soon. Could you please create an issue to keep track of it?

**Comment by xeco23** [comment, marked as answer]
It's implemented now in master with the commit bdfe4113bd2c595c4986fdfa5f483755eb45ddf8
